tests/funcalign/test_srm.py

```diff
@@ -98,7 +98,14 @@ def test_can_instantiate():
             "Invalid computation of SRM! (wrong # features after transform)")
         assert new_s[subject].shape[1] == samples, (
             "Invalid computation of SRM! (wrong # samples after transform)")
-
+    
```

Comment:
Let's add this as part of a new test. 
Add a new test function that creates the object and tries to add the new subject